pylint/pyreverse/vcg_printer.py

```diff
@@ -223,15 +223,38 @@ def emit_node(
         """Create a new node. Nodes can be classes, packages, participants etc."""
         if properties is None:
             properties = NodeProperties(label=name)
+        elif properties.label is None:
+            properties.label = name
         self.emit(f'{self._indent}node: {{title:"{name}"', force_newline=False)
-        label = properties.label if properties.label is not None else name
         self._write_attributes(
             NODE_ATTRS,
-            label=label,
+            label=self._build_label_for_node(properties),
             shape=SHAPES[type_],
         )
         self.emit("}")
 
+    @staticmethod
+    def _build_label_for_node(properties: NodeProperties) -> str:
+        fontcolor = "\f09" if properties.fontcolor == "red" else ""
```

Comment:
I am not particularly happy with hard-coding the "red" here. I am also not sure if the ``\f09`` really changes the color: the only program I could find that can display ``vcg`` graphs does not output the label with a colored text. The documentation of ``vcg`` indicates that it would theoretically also support a ``fontcolor`` attribute.

To be honest I feel like ``vcg`` is not really used any more. As I have a hard time verifying the result due to the lack of a tool which can properly display the graphs I am reluctant to change the implementation of the vcg related code too much.

Reply:
Shouldn't the signature be the same with VcgPrinter and DotPrinter (to respect the Liskov substitution principle) ?

Reply:
Good question - but I would argue no, not in this case:
This method is a "private" method for ``VCGPrinter`` and ``DotPrinter``, but it is intentionally not a part of the ``Printer`` base class. 
For Dot and VCG I need to put the attributes and methods of a class inside the ``label`` attribute for the node, while for the upcoming ``PlantUmlPrinter`` this is not necessary as the syntax is different.
So ``PlantUmlPrinter`` won't have such a ``_build_label_for_node`` method. 
``VCGPrinter`` and ``DotPrinter`` just somewhat "coincidentally" share a very similar method.

Reply:
Make sense 👍